spring-boot-starter-camunda-connectors/src/main/java/io/camunda/connector/runtime/inbound/webhook/InboundWebhookRestController.java

```diff
@@ -71,7 +76,8 @@ public InboundWebhookRestController(
   public ResponseEntity<WebhookResponse> inbound(
       @PathVariable String context,
       @RequestBody(required = false) byte[] bodyAsByteArray, // raw form required to calculate HMAC
-      @RequestHeader Map<String, String> headers)
+      @RequestHeader Map<String, String> headers,
+      @RequestHeader(value = "Content-type", required = false) String contentType)
```

Comment:
Let's pull it from the map `headers` from above?

```suggestion
      @RequestHeader Map<String, String> headers)
```